ai-ml/aibrix/examples/gradio-ui.yaml

```diff
@@ -0,0 +1,88 @@
+# gradio-deploy.yaml
```

Comment:
Would it be possible to leverage [Open Web UI](https://openwebui.com/) instead of Gradio ? We have examples for this and it's under Deepseek pattern